### ISSUE TITLE
issue link status accessibility issues

### ISSUE THREAD

**Issue by jnurthen** [w3c/respec#4172]
## Description of problem

* Issue links `<p class="issue" data-number="1746">` appear the same to screen readers whether they are open or closed
* Color contrast of issue labels is not always sufficient

**URL to affected spec or repo:**

Cannot use this feature until the accessibility issues are resolved

**What happened (e.g., it crashed)?:**

* Closed issues are indistinguishable from open issues
* Issue labels use black text even on a dark background. 

**Expected behavior (e.g., it shouldn't crash):**

* Screen readers should be able to tell if the issue is closed or not - There is a strikethrough visually but nothing for screen readers
* Should be able to read issue labels

**Optional, steps to reproduce:**

* Add a closed issue and an open issue to a spec
* Turn on a screen reader
* Read the page
<img width="841" alt="Screen Shot of 2 issues with a label - one open and one closed" src="https://user-images.githubusercontent.com/1724479/170315375-7552a5b0-5b75-442b-b973-26decdd15d5f.png">
